lib/dev/ui/interactive_prompt.rb

```diff
@@ -38,20 +38,15 @@ def initialize(options)
       # Usually used from +self.call+
       #
       def call
-        Dev::UI.raw { print(ANSI.hide_cursor) }
         while @answer.nil?
           render_options
           wait_for_user_input
-
-          # This will put us back at the beginning of the options
-          # When we redraw the options, they will be overwritten
-          Dev::UI.raw do
-            num_lines = @options.join("\n").split("\n").reject(&:empty?).size
-            num_lines.times { print(ANSI.previous_line) }
-            print(ANSI.previous_line + ANSI.end_of_line + "\n")
-          end
+          reset_position
         end
-        render_options
+
+        clear_output
+        reset_position
```

Comment:
Can `clear_ouput ` finish by calling `reset_position`?  I'd expect it to, and can't imaging a circumstance where I wouldn't want it to.